spec/models/volunteer_spec.rb

```diff
@@ -114,6 +114,62 @@
     end
   end
 
+  describe '#region_names' do
+    let(:boulder) { create(:region, name: 'Boulder') }
+    let(:denver) { create(:region, name: 'Denver') }
+    let(:volunteer_with_shifts) { create(:volunteer, assigned: true) }
+    let(:schedule_chain) { create(:schedule_chain, region: boulder) }
+    let(:shiftless_volunteer) { create(:volunteer, assigned: true) }
+    let(:shiftless_volunteer_denver) { create(:volunteer, regions: [denver], assigned: true) }
+
+    before do
+      create(:assignment, volunteer: volunteer_with_shifts, region: boulder)
+      create(:schedule_volunteer, volunteer: volunteer_with_shifts, schedule_chain: schedule_chain)
+      create(:assignment, volunteer: shiftless_volunteer, region: boulder)
+      create(:assignment, volunteer: shiftless_volunteer_denver, region: denver)
+    end
+
+    it 'returns region name' do
+      expect(volunteer_with_shifts.region_names).to eq('Boulder')
```

Comment:
Is it worth testing that someone in two regions gets it correctly displayed? How about someone in no regions?

Reply:
Always! Thanks for the pointer. 

How does it look to you now?